proto/tendermint/types/params.pb.go

```diff
@@ -591,6 +660,30 @@ func (this *HashedParams) Equal(that interface{}) bool {
 	}
 	return true
 }
+func (this *ABCIParams) Equal(that interface{}) bool {
+	if that == nil {
+		return this == nil
+	}
+
+	that1, ok := that.(*ABCIParams)
+	if !ok {
+		that2, ok := that.(ABCIParams)
+		if ok {
+			that1 = &that2
+		} else {
+			return false
+		}
+	}
+	if that1 == nil {
```

Comment:
can `that` pass the test in 664 and we still have `that1` be `nil` here?

Reply:
Hi Lásaro, this is autogenerated code from protobufs, so we can't really change it
(we commit the autogenerated code because we chose not to require end users to install the toolchain to generate the code at every build)